pixels-index/pixels-index-rocksdb/pom.xml

```diff
@@ -115,6 +115,12 @@
             <version>2.0.3</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.16.1</version>
```

Comment:
Is this dependency used in the code? If it is really needed, please add this dependency to the <dependencyManagement> domain in the root pom.xml, taking the other dependencies as examples.